frontend/model/state.js

```diff
@@ -220,9 +246,9 @@ const getters = {
       .filter(contractID => contracts[contractID].type === 'group' && state[contractID].settings)
       .map(contractID => ({ groupName: state[contractID].settings.groupName, contractID }))
   },
-  memberProfile (state, getters) {
-    return (username, groupId) => {
-      var profile = state[groupId || state.currentGroupId].profiles[username]
+  memberProfile (state) {
+    return (username) => {
+      var profile = state[state.currentGroupId].profiles[username]
```

Comment:
why can't we pass groupId here?

Reply:
because right now getters cannot be access from the contract safely, and i'm working on making it possible to do that... i'll explain in more detail offline